libmcount/mcount.c

```diff
@@ -32,7 +32,9 @@
 #include "utils/symbol.h"
 #include "utils/filter.h"
 #include "utils/compiler.h"
+#ifdef HAVE_LIBPYTHON2
 #include "utils/script.h"
+#endif
```

Comment:
I'd like to remove #ifdef in source file if possible.  Common technique in the kernel code is defining a dummy function in the header (with #ifdef of course).  For example, you could keep utils/script.h but its functions return immediately if python support is not enabled.  Then source can simply call the functions and compiler can optimize it out.